src/components/menus/Items.ts

```diff
@@ -114,6 +115,27 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
         const options: any[] = [
             {
                 callback: (): void => {
```

Comment:
Send this `"GIVE"` code as a separate pull request. I can take a look at why the menu isn't working properly.